.rubocop_todo.yml

```diff
@@ -971,6 +971,7 @@ Style/PerlBackrefs:
 Style/PredicateName:
   Exclude:
     - 'spec/**/*'
+    - 'src/api/app/controllers/webui/kiwi/images_controller.rb'
```

Comment:
@mschnitzer 

> For pundit makes sense for sure, in controllers maybe not that much, at least the ? part. Rubocop doesn't care about Rails, is just a static ruby code analyzer..., so for us is maybe better to skip the rule in controllers or in the place it happens. An action called outdated? is a little bit strange but if the method is not an action then maybe is ok.

Yes, that was exactly the discussion I had in issue I post you before, that it could make sense to disable this for controllers. But this cop is not enabled in our configuration, that why this is not done yet.